client/src/app/auth/firebase-auth.service.ts

```diff
@@ -2,9 +2,7 @@ import * as firebase from 'firebase';
 
 import { Injectable } from '@angular/core';
 
-import { Observable } from 'rxjs/Observable';
-import { fromPromise } from 'rxjs/observable/fromPromise';
-import { of } from 'rxjs/observable/of';
+import { Observable, from as fromPromise, of } from 'rxjs';
```

Comment:
Just wondering why we wouldn't go for just `from` and change the code?

Reply:
This was done by `rxjs-tslint` so I just went with it.